src/core/CCOpt.mli

```diff
@@ -23,119 +23,126 @@ val is_none : _ t -> bool
 
 val flat_map : ('a -> 'b t) -> 'a t -> 'b t
-(** Flip version of {!>>=}. *)
+(** [flat_map f o] is equivalent to {!map} followed by {!flatten}.
+    Flip version of {!>>=}. *)
 
 val bind : 'a t -> ('a -> 'b t) -> 'b t
-(** Monadic bind.
-    [bind f o] if [o] is [Some v] then [f v] else [None]
+(** [bind o f] is [f v] if [o] is [Some v], [None] otherwise.
+    Monadic bind.
     @since NEXT_RELEASE *)
 
 val (>>=) : 'a t -> ('a -> 'b t) -> 'b t
-(** Infix version of {!bind}. *)
+(** [o >>= f] is the infix version of {!bind}. *)
 
 val map2 : ('a -> 'b -> 'c) -> 'a t -> 'b t -> 'c t
 (** [map2 f o1 o2] maps ['a option] and ['b option] to a ['c option] using [f]. *)
 
 val iter : ('a -> unit) -> 'a t -> unit
-(** Iterate on 0 or 1 element. *)
+(** [iter f o] applies [f] to [o]. Iterate on 0 or 1 element. *)
 
 val fold : ('a -> 'b -> 'a) -> 'a -> 'b t -> 'a
-(** Fold on 0 or 1 element. *)
+(** [fold f init o] is [f init x] if [o] is [Some x], or [init] if [o] is [None].
+    Fold on 0 or 1 element. *)
 
 val filter : ('a -> bool) -> 'a t -> 'a t
-(** Filter on 0 or 1 element.
+(** [filter f o] returns [Some x] if [f (Some x)] is [true],
+    or [None] if [f (Some x)] is [false] or if [o] is [None].
+    Filter on 0 or 1 element.
     @since 0.5 *)
 
 val if_ : ('a -> bool) -> 'a -> 'a option
 (** [if_ f x] is [Some x] if [f x], [None] otherwise.
     @since 0.17 *)
 
 val exists : ('a -> bool) -> 'a t -> bool
-(** Return [true] iff there exists an element for which the provided function evaluates to [true].
+(** [exists f o] returns [true] iff there exists an element for which
+    the provided function [f] evaluates to [true].
     @since 0.17 *)
 
 val for_all : ('a -> bool) -> 'a t -> bool
-(** Return [true] iff the provided function evaluates to [true] for all elements.
+(** [for_all f o] returns [true] iff the provided function [f] evaluates to [true] for all elements.
     @since 0.17 *)
 
 val get_or : default:'a -> 'a t -> 'a
 (** [get_or ~default o] extracts the value from [o], or
-    returns [default] if [o = None].
+    returns [default] if [o] is [None].
     @since 0.18 *)
 
 val value : 'a t -> default:'a -> 'a
-(** Similar to the stdlib's [Option.value] and to {!get_or}.
+(** [value o ~default] is similar to the Stdlib's [Option.value] and to {!get_or}.
     @since 2.8 *)
 
 val get_exn : 'a t -> 'a
-(** Open the option, possibly failing if it is [None].
+(** [get_exn o] returns [x] if [o] is [Some x] or fails if [o] is [None].
     @raise Invalid_argument if the option is [None]. *)
 
 val get_lazy : (unit -> 'a) -> 'a t -> 'a
-(** [get_lazy default_fn x] unwraps [x], but if [x = None] it returns [default_fn ()] instead.
+(** [get_lazy default_fn o] unwraps [o], but if [o] is [None] it returns [default_fn ()] instead.
     @since 0.6.1 *)
 
 val sequence_l : 'a t list -> 'a list t
-(** [sequence_l [x1; x2; ...; xn]] returns [Some [y1;y2;...;yn]] if
+(** [sequence_l [x1; x2; ⋯; xn]] returns [Some [y1; y2; ⋯;yn]] if
```

Comment:
do you mean to use `…` instead of `···` ? :)

Reply:
Euh, yes I mean to use … instead of . . . 
Is there a pb somewhere?

Note: maybe ⋯ would be better than ⋯; (no ; after ⋯)?

Reply:
you're using · instead of . in your dots (middle of the line instead of bottom).

Reply:
OK, I see.
Killing all these little beasts.